dali/test/python/test_video_pipeline.py

```diff
@@ -216,19 +223,32 @@ def _test_file_list_empty_videopipeline(start, end):
     list_file.close()
 
     pipe = VideoPipeList(batch_size=BATCH_SIZE, data=list_file.name)
-    assert_raises(RuntimeError, pipe.build)
+    with assert_raises(RuntimeError, glob='*Start frame number should be lesser or equal to end frame number for a file*'):
+        pipe.build()
     os.remove(list_file.name)
 
-def test_file_list_empty_videopipeline():
+
+def test_file_list_invalid_range():
     invalid_ranges = [
-        [0, 0],
-        [10, 10],
         [-1, 1],
         [1000000, None],
         [0, -1000]
     ]
     for r in invalid_ranges:
-        yield _test_file_list_empty_videopipeline, r[0], r[1]
+        yield _test_file_list_invalid_range, r[0], r[1]
+
+
+def test_file_list_empty():
+    files = sorted(os.listdir(VIDEO_DIRECTORY))
+    list_file = tempfile.NamedTemporaryFile(mode="w", delete=False)
+    list_file.write("{} {} {} {}\n".format(os.path.join(VIDEO_DIRECTORY, files[0]), 0, 10, 10))
+    list_file.close()
+
+    pipe = VideoPipeList(batch_size=BATCH_SIZE, data=list_file.name)
+    with assert_raises(RuntimeError, glob='*No files were read'):
```

Comment:
I would improve video_loader error message in this case as we discussed offline.

Reply:
As we agreed, the warning  about empty range, that is already there,  should be enough